test/test-applications/integrations/TestApplication.GraphQL/CsrfMiddleware.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+namespace TestApplication.GraphQL;
+
+public class CsrfMiddleware
```

Comment:
Could add some words what is the exact purpose of this middleware.

Reply:
Done